src/layout/RepeatingGroup/RepeatingGroupContainer.tsx

```diff
@@ -57,10 +61,13 @@ export const RepeatingGroupContainer = forwardRef((_, ref: React.ForwardedRef<HT
       data-componentbaseid={node.baseId}
       ref={ref}
     >
-      {(!edit?.mode ||
-        edit?.mode === 'showTable' ||
-        edit?.mode === 'onlyTable' ||
-        (edit?.mode === 'hideTable' && !isEditingAnyRow)) && <RepeatingGroupTable />}
+      {showTable && <RepeatingGroupTable />}
+      {!showTable && textResourceBindings?.title && (
+        <Label
+          renderLabelAs='span'
```

Comment:
Burde labelen her være en `fieldset`/`legend` siden den er for en tabell?

Reply:
Er ingen ekspert selv, men så vidt jeg har skjønt så skal `fieldset` med `legend` brukes når det er en label for flere elementer.
https://html.spec.whatwg.org/multipage/form-elements.html#the-fieldset-element

Reply:
Allright, jeg skal se om det kan ordnes uten veldig mye krøll. :raised_hands: